jupyter_client/multikernelmanager.py

```diff
@@ -104,7 +104,6 @@ def start_kernel(self, kernel_name=None, **kwargs):
         km = self.kernel_manager_factory(connection_file=os.path.join(
                     self.connection_dir, "kernel-%s.json" % kernel_id),
                     parent=self, log=self.log, kernel_name=kernel_name,
-                    **kwargs
```

Comment:
But it's also passing through to `start_kernel()` just below. Is there any way to define the arguments that either call should take, so we can filter them out?